CONTRIBUTING.md

```diff
@@ -117,14 +117,32 @@ See the [scala-jenkins-infra repo](https://github.com/scala/scala-jenkins-infra)
 
 ### Pass code review
 
-Your PR will need to be assigned to one or more reviewers.  You can suggest reviewers yourself; if you're not sure, see the list in [README.md](README.md) or ask on scala-internals.
+Your PR will need to be assigned to one or more reviewers. You can suggest reviewers
+yourself; if you're not sure, see the list in [README.md](README.md) or ask on gitter
```

Comment:
i don't think so, check the preview https://github.com/lrytz/scala/blob/c8b618d616349d3515888d8d85e85689bfae7285/CONTRIBUTING.md